homework/index.js

```diff
@@ -37,41 +37,35 @@
     container.innerHTML = '';
     createAndAppend('div', container, {
       text: error.message,
-      class: 'error',
+      class: 'alert-error',
     });
   }
 
+  function addRow(table, text, value) {
+    const tableRow = createAndAppend('tr', table);
+    createAndAppend('th', tableRow, {
+      text: text,
```

Comment:
This produces an ESLint error and therefore a Travis error, but it is not your fault. In modern (ES6) JavaScript you can use 'short-hand' notation in cases where an object property name is the same as the variable name you assign to it. This short-hand notation is generally preferred and the (Airbnb) ESLint rules have been set up to complain about it if you don't use it. It is a **_style_** error, not a JavaScript error. 

With the short-hand notation, your code would look like this:

```js
createAndAppend('th', tableRow, {
  text,
  class: 'table-header',
});
```